lib/renderer/api/remote.js

```diff
@@ -340,9 +309,7 @@ exports.__defineGetter__('process', function () {
 
 // Create a funtion that will return the specifed value when called in browser.
 exports.createFunctionWithReturnValue = function (returnValue) {
-  const func = function () {
```

Comment:
Note that arrow functions implicitly bind `this`. Not sure if it matters here, but it could have bad consequences. Something to keep in mind!

Reply:
thanks! i think it's fine in this case since `this` isn't involved but agree, definitely just can't blindly replace functions with arrow functions